services/lsst-dev.rst

```diff
@@ -27,16 +27,13 @@ This page is designed to assist developers in their work on the ``lsst-dev`` ser
 Overview of Cluster Resources
 =============================
 
-- List of `available development servers <https://confluence.lsstcorp.org/display/LDMDG/DM+Development+Servers>`_ and their intended use.
-- `Service status <https://confluence.lsstcorp.org/display/DM/LSST+Service+Status+page>`_ including announcements of upcoming planned down-time.
-- `Real-time system status <https://monitor-ncsa.lsst.org/>`_ (requires login).
+Refer to :ref:`lsst-login-overview` for a general overview of LDF cluster resources.
+
 - All of the ``lsst-dev`` systems have access to the :ref:`verification-gpfs`, including:
```

Comment:
Since there's nothing else listed here, I don't think this needs to be bulleted.